scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -785,6 +762,57 @@ async def start_reborn_server(
+        async with httpx.AsyncClient(timeout=30.0) as client:
+            response = await client.put(
+                f"{base_url}/api/webchat/v2/channels/slack/setup",
+                headers={"Authorization": f"Bearer {AUTH_TOKEN}"},
+                json=payload,
+            )
+            response_text = response.text[:1000]
+            if response.status_code < 200 or response.status_code >= 300:
+                raise LiveQaError(
+                    "Slack setup API returned HTTP "
+                    f"{response.status_code}: {response_text!r}"
+                )
```

Ruff (0.15.20)

[warning] Abstract `raise` to an inner function

(TRY301)